### PR TITLE
Adding Webrtc VAD

### PR DESCRIPTION
**Pull request by andrenatal**

### CODE REVIEW

**Comment by andrenatal** [comment]
@fabricedesre Could you please review? 

Please not that I merged  webrtc_vad.js into content.js to move on with the VAD algorithm integration in the WE until we figure out how can we load them always in sequence using two different js files.

File: extension/content.js [hunk 1]
```diff
@@ -216,71 +215,78 @@ const on_spm_icon_click = (event) => {
...
+            // VAD initializations
+            // console.log("Sample rate: ", audioContext.sampleRate);
+            let bufferSize = 2048;
+            //create a javascript node
+            let scriptprocessor = audioContext.createScriptProcessor(bufferSize, 1, 1);
+            // specify the processing function
+            stm_vad.reset();
+            scriptprocessor.onaudioprocess = stm_vad.recorderProcess;
+            stm_vad.stopGum = () => {
+                console.log('stopGum');
```
Comment (fabricedesre) [inline extension/content.js]: nit: This file uses " for string delimiters

File: extension/content.js [hunk 2]
```diff
@@ -339,4 +345,158 @@ SpeakToMePopup.init();
 
 let stm_icon = new SpeakToMeIcon();
 
+// Webrtc_Vad integration
+SpeakToMeVad = function SpeakToMeVad(){
```
Comment (fabricedesre) [inline extension/content.js]: nit: space between () and { (here and for other functions)

File: extension/content.js [hunk 3]
```diff
@@ -339,4 +345,158 @@ SpeakToMePopup.init();
...
+
+    this.webrtc_main = Module.cwrap('main');
+    this.webrtc_main();
+    this.webrtc_setmode = Module.cwrap('setmode', 'number', ['number']);
+    this.webrtc_setmode(3);
+    this.webrtc_process_data = Module.cwrap('process_data', 'number', ['number', 'number', 'number', 'number', 'number', 'number']);
+    this.sizeBufferVad = 480;
```
Comment (fabricedesre) [inline extension/content.js]: That would be nice to add a comment to explain what these parameters are.

File: extension/content.js [hunk 4]
```diff
@@ -339,4 +345,158 @@ SpeakToMePopup.init();
...
+        };
+    })(),
+    printErr: function(text) {
+        console.error("[webrtc_vad.js error]", text);
+    },
+    canvas: (function() {
+    })(),
+    setStatus: function(text) {
+        if (!Module.setStatus.last) Module.setStatus.last = { time: Date.now(), text: '' };
+        if (text === Module.setStatus.text) return;
```
Comment (fabricedesre) [inline extension/content.js]: use { } even for single expression if blocks

File: extension/content.js (hunk 1 above)
Comment (andrenatal) [inline extension/content.js]: Fixed

File: extension/content.js (hunk 2 above)
Comment (andrenatal) [inline extension/content.js]: fixed

File: extension/content.js (hunk 3 above)
Comment (andrenatal) [inline extension/content.js]: Fixed

File: extension/content.js (hunk 4 above)
Comment (andrenatal) [inline extension/content.js]: Fixed

**Comment by andrenatal** [comment]
@fabricedesre , I just made the changes you requested, could you please review again?

**Pull request merged**